link/kprobe_multi_test.go

```diff
@@ -0,0 +1,131 @@
+package link
+
+import (
+	"errors"
+	"math"
+	"os"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+var kprobeMultiSyms = []string{"vprintk", "inet6_release"}
+
+func TestKprobeMulti(t *testing.T) {
+	testutils.SkipOnOldKernel(t, "5.18", "kprobe_multi link")
```

Comment:
Why do we include haveBPFLinkKprobeMulti if the tests don't rely on it? Either strip out the feature test (fine by me!) or use `testutils.SkipIfNotSupported`.